src/twig/components/table/Table.vue

```diff
@@ -279,33 +278,37 @@
                 items: [],
                 currentPage: 1,
                 searchTerm: '',
-                filterTerm: [],
                 filterOptions: [],
                 selectedDepartment: 'All Departments',
                 selectedDivision: 'All Divisions',
+                filters: {
+                    term: 'all',
+                    department: 'all',
+                    division: 'all',
+                },
             };
         },
         computed: {
             filteredItems() {
                 let f = [];
-                if (this.api !== 'Departments') {
+                if (this.api !== 'Departments' && this.api !== 'People' && this.api !== 'Offices') {
```

Comment:
@GioAceto I can't remember. Why did this conditional change so much?

Reply:
In order to get the table filtering to work with People and Offices pages.

Reply:
Ah ok!